.gitignore

```diff
@@ -31,3 +31,4 @@
 
```

Comment:
Thank you for updating the git/ignore file, cleans up the app and would save processing time if this were a more robust application.